src/sage/combinat/k_regular_sequence.py

```diff
@@ -2602,16 +2602,16 @@ def shifted_inhomogeneities(self, recurrence_rules):
             ....:     f(10) == 4, f(11) == 4, f(12) == 12, f(13) == 0, f(14) == 4,
             ....:     f(15) == 4, f(16) == 8, f(17) == 4, f(18) == 8, f(19) == 0,
             ....:     f(20) == 8, f(21) == 4, f(22) == 4, f(23) == 8], f, n, offset=3)
-            sage: inhomogeneities={2: UB.subsequence(4, 3), 3: -UB.subsequence(4, 1),
+            sage: inhomogeneities={2: UB.subsequence(4, 3), 3: -UB.subsequence(4, 1),   # optional - sage.symbolic
             ....:                  6: UB.subsequence(4, 2) + UB.subsequence(4, 3)}
-            sage: recurrence_rules_UB = RR(M=3, m=2, ll=0, uu=9,
+            sage: recurrence_rules_UB = RR(M=3, m=2, ll=0, uu=9,                        # optional - sage.symbolic
             ....:                          inhomogeneities=inhomogeneities)
-            sage: shifted_inhomog = RP.shifted_inhomogeneities(recurrence_rules_UB)
-            sage: shifted_inhomog
+            sage: shifted_inhomog = RP.shifted_inhomogeneities(recurrence_rules_UB)     # optional - sage.symbolic
+            sage: shifted_inhomog                                                       # optional - sage.symbolic
             {2: 2-regular sequence 8, 8, 8, 12, 12, 16, 12, 16, 12, 24, ...,
              3: 2-regular sequence -10, -8, -8, -8, -8, -8, -8, -8, -8, -12, ...,
              6: 2-regular sequence 20, 22, 24, 28, 28, 32, 28, 32, 32, 48, ...}
-            sage: shifted_inhomog[2].mu[0].ncols() == 3*inhomogeneities[2].mu[0].ncols()
+            sage: shifted_inhomog[2].mu[0].ncols() == 3*inhomogeneities[2].mu[0].ncols()    # optional - sage.symbolic
```

Comment:
Starting from this line, the alignment of `# optional... ` is different than previous lines

Reply:
Fixed in #35742 (acf16adb20)